cmd/zsysd/completion.go

```diff
@@ -11,18 +12,58 @@ import (
 func installCompletionCmd(rootCmd *cobra.Command) {
 	prog := rootCmd.Name()
 	var completionCmd = &cobra.Command{
-		Use:   "completion",
-		Short: i18n.G("Generates bash completion scripts"),
-		Long: fmt.Sprintf(i18n.G(`To load completion run
+		Use:   "completion [bash|zsh|powershell]",
+		Short: i18n.G("Generates completion scripts"),
+		Long: strings.ReplaceAll(i18n.G(`To load completions:
 
-. <(%s completion)
+Bash:
 
-To configure your bash shell to load completions for each session add to your ~/.bashrc or ~/.profile:
+  $ source <(#prog# completion bash)
 
-. <(%s completion)
-`), prog, prog),
+  # To load completions for each session, execute once:
+  # Linux:
+  $ #prog# completion bash > /etc/bash_completion.d/#prog#
+  # macOS:
+  $ #prog# completion bash > /usr/local/etc/bash_completion.d/#prog#
+
+Zsh:
+
+  # If shell completion is not already enabled in your environment,
+  # you will need to enable it.  You can execute the following once:
+
+  $ echo "autoload -U compinit; compinit" >> ~/.zshrc
+
+  # To load completions for each session, execute once:
+  $ #prog# completion zsh > "${fpath[1]}/_#prog#"
+
+  # You will need to start a new shell for this setup to take effect.
+
+PowerShell:
+
+  PS> #prog# completion powershell | Out-String | Invoke-Expression
+
+  # To load completions for every new session, run:
+  PS> #prog# completion powershell > #prog#.ps1
+  # and source this file from your PowerShell profile.
+`), "#prog#", prog),
+		DisableFlagsInUseLine: true,
```

Comment:
We still allow other flags like -vv, no?

Reply:
I will remove that.